pages/package.json

```diff
@@ -6,7 +6,6 @@
     "immutable": "^4.0.0-rc.3",
     "prop-types": "^15.6.0",
     "react": "^16.0.0",
-    "react-dom": "^16.0.0",
```

Comment:
Why have you removed `react-dom`?

Reply:
Because i don't use it in the client side.

Reply:
`react-dom` is an implicit dependency, you cannot remove it.

It only works because it's used by an indirect dependency (`react-scripts#react-dev-utils#react-error-overlay`).

Reply:
You have to add this dependency back.